postcss.config.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
-  plugins: [
-    "postcss-import",
-    "tailwindcss",
-    "autoprefixer",
-  ],
+  plugins: {
```

Comment:
This has nothing to do with the purge issue that I'm having so it's better to rollback

Reply:
Done on d0949ae74ba9665102a8a7f0933b66b4099525ce